### ISSUE TITLE
v1.79.0 breaks terraform_providers_lock hook

### ISSUE THREAD

**Issue by z0rc** [antonbabenko/pre-commit-terraform#520]
### Describe the bug

Updating from 1.78.0 to 1.79.0 breaks terraform_providers_lock hook.
### How can we reproduce it?

```sh
% cat .pre-commit-config.yaml
---
repos:
- repo: https://github.com/antonbabenko/pre-commit-terraform
  rev: v1.79.0
  hooks:
  - id: terraform_fmt
  - id: terraform_docs
  - id: terraform_providers_lock
    args:
    - --args=-platform=linux_amd64
    - --args=-platform=linux_arm64
    - --args=-platform=darwin_amd64
    - --args=-platform=darwin_arm64

% pre-commit run -a
Terraform fmt............................................................Passed
Terraform docs...........................................................Passed
Lock terraform provider versions.........................................Failed
- hook id: terraform_providers_lock
- exit code: 1

╷
│ Error: Invalid provider argument
│
│ The provider registry.terraform.io/hashicorp/false is not required by the current configuration.
╵

╷
│ Error: Invalid provider type
│
│ Invalid provider type "-platform=linux_amd64" in source "-platform=linux_amd64": must contain only letters, digits, and dashes, and may not use leading or trailing dashes"
╵

╷
│ Error: Invalid provider type
│
│ Invalid provider type "-platform=linux_arm64" in source "-platform=linux_arm64": must contain only letters, digits, and dashes, and may not use leading or trailing dashes"
╵

╷
│ Error: Invalid provider type
│
│ Invalid provider type "-platform=darwin_amd64" in source "-platform=darwin_amd64": must contain only letters, digits, and dashes, and may not use leading or trailing dashes"
╵

╷
│ Error: Invalid provider type
│
│ Invalid provider type "-platform=darwin_arm64" in source "-platform=darwin_arm64": must contain only letters, digits, and dashes, and may not use leading or trailing dashes"
╵
```

Reverting to v1.78.0 allows hook to pass without issue.

Maybe it is worth noting that terraform code is located in subfolder of git repo, like `<git_repo>/infra`.

### Environment information

* OS: MacOS and Ubuntu 22.04
* Tools availability and versions:
```bash
% terraform --version
Terraform v1.4.6
on darwin_arm64
```

**Comment by MaxymVlasov** [comment]
Silly error, will be fixed in 30 min

**Comment by antonbabenko** [comment]
This issue has been resolved in version 1.79.1 :tada:
